melodies_monet/util/sat_l2_swath_utility_tempo.py

```diff
@@ -0,0 +1,985 @@
+# Copyright (C) 2024 National Center for Atmospheric Research and National Oceanic and Atmospheric Administration
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
This header is failing the check because of the different year. Use 2022 for consistency or we could update the check (but maybe check with @rschwant and Louisa first).

Reply:
I'll wait for their answer for this, then